c/parallel/api.h

```diff
@@ -101,7 +112,7 @@ void barrier();
  * Run parallel loop `for i in range(nrows): f(i)`, with dynamic scheduling.
  */
 void parallel_for_dynamic(size_t nrows, std::function<void(size_t)> fn);
-void parallel_for_dynamic(size_t nrows, size_t nthreads,
+void parallel_for_dynamic(size_t nrows, NThreads _NThreads,
```

Comment:
I don't think that's a valid variable name in C++, see https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

Reply:
Changed it everywhere to `NThreads_` to be safe.